composer.json

```diff
@@ -90,51 +90,50 @@
-        "symfony/expression-language": "^5.4.21 || ^6.0",
-        "symfony/filesystem": "^5.4.21 || ^6.0",
-        "symfony/finder": "^5.4.21 || ^6.0",
-        "symfony/form": "^5.4.21 || ^6.0",
-        "symfony/framework-bundle": "^5.4.21 || ^6.0",
-        "symfony/http-client": "^5.4.21 || ^6.0",
-        "symfony/http-foundation": "^5.4.21 || ^6.0",
-        "symfony/http-kernel": "^5.4.21 || ^6.0",
-        "symfony/intl": "^5.4.21 || ^6.0",
-        "symfony/mailer": "^5.4.21 || ^6.0",
-        "symfony/messenger": "^5.4.21 || ^6.0",
-        "symfony/monolog-bundle": "^3.5",
-        "symfony/options-resolver": "^5.4.21 || ^6.0",
-        "symfony/password-hasher": "^5.4.21 || ^6.0",
-        "symfony/polyfill-iconv": "^1.17",
-        "symfony/polyfill-intl-icu": "^1.22",
-        "symfony/polyfill-mbstring": "^1.17",
-        "symfony/polyfill-php80": "^1.17",
-        "symfony/process": "^5.4.21 || ^6.0",
-        "symfony/property-access": "^5.4.21 || ^6.0",
-        "symfony/property-info": "^5.4.21 || ^6.0",
-        "symfony/proxy-manager-bridge": "^5.4.21 || ^6.0",
-        "symfony/routing": "^5.4.21 || ^6.0",
-        "symfony/security-bundle": "^5.4.21 || ^6.0",
-        "symfony/security-core": "^5.4.21 || ^6.0",
-        "symfony/security-csrf": "^5.4.21 || ^6.0",
-        "symfony/security-http": "^5.4.21 || ^6.0",
-        "symfony/serializer": "^5.4.21 || ^6.0",
+        "symfony/asset": "^6.3.0",
+        "symfony/config": "^6.3.2",
+        "symfony/console": "^6.3.4",
+        "symfony/dependency-injection": "^6.3.4",
+        "symfony/deprecation-contracts": "^2.5.2",
+        "symfony/doctrine-bridge": "^6.3.4",
+        "symfony/doctrine-messenger": "^6.3.1",
+        "symfony/event-dispatcher": "^6.3.2",
+        "symfony/expression-language": "^6.3.0",
+        "symfony/filesystem": "^6.3.1",
+        "symfony/finder": "^6.3.3",
+        "symfony/form": "^6.3.2",
+        "symfony/framework-bundle": "^6.3.4",
+        "symfony/http-client": "^6.3.2",
+        "symfony/http-foundation": "^6.3.4",
+        "symfony/http-kernel": "^6.3.4",
+        "symfony/intl": "^6.3.2",
+        "symfony/mailer": "^6.3.0",
+        "symfony/messenger": "^6.3.4",
+        "symfony/monolog-bundle": "^3.8.0",
+        "symfony/options-resolver": "^6.3.0",
+        "symfony/password-hasher": "^6.3.0",
+        "symfony/polyfill-iconv": "^1.28",
+        "symfony/polyfill-intl-icu": "^1.28",
+        "symfony/polyfill-mbstring": "^1.28",
+        "symfony/process": "^6.3.4",
+        "symfony/property-access": "^6.3.2",
+        "symfony/property-info": "^6.3.0",
+        "symfony/proxy-manager-bridge": "^6.3.0",
+        "symfony/routing": "^6.3.3",
+        "symfony/security-bundle": "^6.3.4",
+        "symfony/security-core": "^6.3.3",
+        "symfony/security-csrf": "^6.3.2",
+        "symfony/security-http": "^6.3.4",
+        "symfony/serializer": "^6.3.4",
         "symfony/service-contracts": "^2.5 || ^3.0",
```

Comment:
It should be possible just by looking at packagist, something blocking it? (There are other instances of this dependency in other composer.json)

```patch
--- "symfony/service-contracts": "^2.5 || ^3.0",
+++ "symfony/service-contracts": "^3.3",
```

Reply:
```
sylius/sylius                 v1.13.0-dev requires  symfony/service-contracts (^2.5 || ^3.0)         
doctrine/doctrine-bundle      2.10.2      requires  symfony/service-contracts (^1.1.1 || ^2.0 || ^3) 
sylius/theme-bundle           v2.3.0      requires  symfony/service-contracts (^1.1 || ^2.0 || ^3.0) 
symfony/cache                 v5.4.28     requires  symfony/service-contracts (^1.1|^2|^3)           
symfony/console               v5.4.28     requires  symfony/service-contracts (^1.1|^2|^3)           
symfony/dependency-injection  v5.4.28     requires  symfony/service-contracts (^1.1.6|^2)            
symfony/doctrine-bridge       v5.4.28     requires  symfony/service-contracts (^1.1|^2|^3)           
symfony/doctrine-messenger    v5.4.25     requires  symfony/service-contracts (^1.1|^2|^3)           
symfony/expression-language   v5.4.21     requires  symfony/service-contracts (^1.1|^2|^3)           
symfony/form                  v5.4.26     requires  symfony/service-contracts (^1.1|^2|^3)           
symfony/framework-bundle      v5.4.28     conflicts symfony/service-contracts (>=3.0)                
symfony/http-client           v5.4.26     requires  symfony/service-contracts (^1.0|^2|^3)           
symfony/mailer                v5.4.22     requires  symfony/service-contracts (^1.1|^2|^3)           
symfony/monolog-bridge        v5.4.22     requires  symfony/service-contracts (^1.1|^2|^3)           
symfony/security-core         v5.4.22     requires  symfony/service-contracts (^1.1.6|^2|^3)         
symfony/stopwatch             v5.4.21     requires  symfony/service-contracts (^1|^2|^3)             
symfony/twig-bundle           v5.4.27     conflicts symfony/service-contracts (>=3.0)                
symfony/webpack-encore-bundle v1.17.1     requires  symfony/service-contracts (^1.0 || ^2.0 || ^3.0) 

```